demo/components/Playground.tsx

```diff
@@ -185,6 +187,9 @@ export const Playground = React.memo<PlaygroundProps>((props) => {
             needToSetDimensionsForUploadedImages,
             renderPreview: renderPreviewDefined ? renderPreview : undefined,
             fileUploadHandler,
+            md: {
+                emptyRowPlaceholder: emptyRowPlaceholder,
+            },
```

Comment:
I propose some changes:
1. move setting from `md` to `wysiwygConfig`
2. in `wysiwygConfig` add new optional property for placeholder configuration, that have structure like that:
  ```
  value: string | () => string;
  behavior: 'empty-doc' | 'empty-row'
  ```